documentation/staging/content/quickstart/prepare.md

```diff
@@ -24,7 +24,30 @@ weight: 5
         --set "kubernetes.namespaces={traefik,sample-domain1-ns}"
     ```
 
-{{% notice note %}}
-If you have reached this point while following the "Model in Image" sample, please
-stop here and return to the [sample instructions]({{< relref "/samples/domains/model-in-image/prerequisites#resume" >}}).
-{{% /notice %}}
+    {{% notice note %}} If you have reached this point while following the "Model in Image" sample, please
+    stop here and return to the [sample instructions]({{< relref "/samples/domains/model-in-image/prerequisites#resume" >}}).
+    {{% /notice %}}
+
+1. Accept the license agreement for WebLogic Server images.
+
+    a. In a browser, go to the [Oracle Container Registry](https://container-registry.oracle.com/) (OCR) and
+    log in using the Oracle Single Sign-On (SSO) authentication service. If you do not already have SSO credentials,
+    then at the top, right side of the page, click Sign In to create them.
+
+    b. Search for `weblogic`, then select `weblogic` in the Search Results.
+
+    c. From the drop-down menu, select your language and click Continue.
+
+    d. Then, read and accept the license agreement.
+
+1. Create a `docker-registry` secret to enable pulling the example image from the registry.
```

Comment:
example image --> ?

I assume the aux image _does not_ also need the registry secret? @ankedia would know. 

If it _does_, then make it "example WebLogic Server image and sample auxiliary image" or "example images".

If it _does not_, then make it "example WebLogic Server image" to disambiguate from the aux image.